File: Pascal/1020/1020.pas
var
	n : integer;
	days, months, years : integer;
begin
	readln(n);

	years := n div 365;
	months := n mod 365 div 30;
	days := n mod 365 mod 30;

	writeln(years, ' ano(s)'#10, months, ' mes(es)'#10, days, ' dia(s)');
	{#10 or #13#10 is the same as \n in other language}
end.

File: C++/1020/1020.cpp
#include <iostream>
#include <cstdio>
using namespace std;

int main() {
		int ages, years, months, days;

		cin >> ages;
		
		years = ages / 365;
		months = ages % 365 / 30;
		days = ages % 365 % 30;

		printf("%i ano(s)\n%i mes(es)\n%i dia(s)\n", years, months, days);
}
